dart-impl/mpi/include/dash/dart/mpi/dart_communication_priv.h

```diff
@@ -79,7 +79,24 @@ dart__mpi__datatype_init() DART_INTERNAL;
 dart_ret_t
 dart__mpi__datatype_fini() DART_INTERNAL;
 
+
+// TODO: types
+DART_INLINE void dart__min_max_reduce(void *lhs_, void *rhs_, int *len, MPI_Datatype *dptr)
+{
+  float *lhs = (float *)lhs_;
```

Comment:
I would eliminate these magic numbers and replace them with macro definition. The code then like this:
```c
if (rhs[MIN_IDX] > lhs[MIN_IDX]) {
  rhs[MIN_IDX] = lhs[MIN_IDX];
}
```